LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2014, Rod Vagg (the "Original Author")
+Copyright 2013-2015, Rod Vagg (the "Original Author")
```

Comment:
assignment needs to be fixed here too, have a look at the LICENSE.md (note markdown!) in levelup, we can just copy that across.